data/lm/README.rst

```diff
@@ -1,32 +1,23 @@
-Download librispeech corpus
+The LM binary was generated from the LibriSpeech normalized LM training text, available `here <http://www.openslr.org/11>`_.
+It is created with `KenLM <https://github.com/kpu/kenlm>`_.
+
+You can download the librispeech corpus with the following commands:
 
 .. code-block:: bash
 
     wget http://www.openslr.org/resources/11/librispeech-lm-norm.txt.gz -O librispeech.txt.gz
     gunzip librispeech.txt.gz
```

Comment:
nit: Remove.

Reply:
Delete the intro text again?

Reply:
No no, just the `gunzip` part since the script can now read from the .gz version.